Client/Ecosia/Extensions/AppDelegate.Ecosia.swift

```diff
@@ -12,6 +12,7 @@ extension AppDelegate {
         Task {
             do {
                 let env: Environment = AppConstants.BuildChannel == .release ? .production : .staging
+                Environment.current = env
```

Comment:
I guess this is the fix for what you mentioned:
> When testing the Unleash config retrieval, I've noticed an issue when .prod build chanell was always sent.

At what point do we change to staging that was not reflecting here? Just wondering since I'm not sure I totally get the issue 🤔

Reply:
Good question. The point at which I realized the `staging` parameter was not passed in the Unleash proxy BE call. And I was worried I was not getting the expected values.
Upon creating the request in the Core library, we `Environment` retrieved is `prod`.

```
(lldb) po Environment.current.unleash
"prod"
```

Hence, the query parameter passed in the request is `environment=prod`

Interestingly, the `mob_ios_bing_market` parameter is still retrieved with the expected value set to `staging`.

So I'm going to remove this line as I'm not 💯 sure it has no other implications, but logically, the behavior shouldn't be the above 😅 .